draft-js-resizeable-plugin/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+### Fixed
+- fixed incorrect behavior when resized from the left edge [#582](https://github.com/draft-js-plugins/draft-js-plugins/issues/582)
+
```

Comment:
instead of adding `fixed` you can increment the version, just like below, since we need to publish this under `2.0.3`

btw, mind also bumping the version of the resizable plugin in the package.json? Then I can just run the script and everything will be published accordingly

Reply:
My pleasure